### PR TITLE
Rename Tendermint to Cosmos

### PR DESCRIPTION
**Pull request by pienkowb**

### CODE REVIEW

File: src/protocols/cosmos/scaffold/manifest.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const source = () => `
+      startBlock: 0`
```
Comment (evaporei) [inline src/protocols/cosmos/scaffold/manifest.js:2]: Isn't `startBlock` zero by default? 🤔

Comment (pienkowb) [reply src/protocols/cosmos/scaffold/manifest.js:2]: So you're suggesting to remove `startBlock` entirely (as it's `0` by default)?

Comment (evaporei) [reply src/protocols/cosmos/scaffold/manifest.js:2]: Yes!

**Review by evaporei — APPROVED** [review]

**Comment by evaporei** [comment]
@pienkowb it seems you'll need to solve conflicts with the `main` branch.

**Pull request merged**